proof/drefine/Arch_DR.thy

```diff
@@ -1440,15 +1440,15 @@ lemma invoke_page_directory_corres:
   apply (clarsimp simp: perform_page_directory_invocation_def)
   done
 
-lemma pte_check_if_mapped_corres:
+lemma pteCheckIfMapped_corres:
   "dcorres dc \<top> \<top> (return a) (pte_check_if_mapped pte)"
```

Comment:
Whoops, I didn't catch these last time. This is similar to the `spec/capDL` issue. DRefine goes up from the abstract spec, so we're refining from something even more abstract down to `pte_check_if_mapped` from the abstract spec. Here, the even-more-abstract function is just `return a`, since the DSpec throws away quite a lot of information, including whether pages are mapped. So there's definitely no `pteCheckIfMapped` function involved here.

You should look through all the changes you've made in `proof/drefine`. I think you'll find that you just need to revert everything in this directory.

Reply:
`proof/drefine` changes rolled back! Will need to remember to remove commit message when squashing for the merge to `master`